mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.kt

```diff
@@ -57,7 +57,8 @@ class BackgroundTasksManager : BroadcastReceiver() {
                 NotificationUpdateObserver.createNotificationChannels(context)
             }
             ACTION_RETRY_UPLOAD -> {
-                for (info in intent.getParcelableArrayListExtra<RetryInfo>(EXTRA_RETRY_INFO_LIST)) {
+                val retryInfoList = intent.getParcelableArrayListExtra<RetryInfo>(EXTRA_RETRY_INFO_LIST) ?: return
+                for (info in retryInfoList) {
```

Comment:
Is there something like `for (info in? retryInfoList)`?

Reply:
Maybe `retryInfoList?.forEach { info ->`?